.gitignore

```diff
@@ -0,0 +1 @@
+.php_cs.cache
```

Comment:
This line should be in your gitignore global

Reply:
I have already put it there